src/Platform.Handlers/src/Xamarin.Platform.Handlers/Platform/Android/ProgressBarExtensions.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Android.Content.Res;
+using Android.OS;
+using Xamarin.Forms;
+using AProgressBar = Android.Widget.ProgressBar;
+
+namespace Xamarin.Platform
+{
+	public static class ProgressBarExtensions
+	{
+		public static void UpdateProgress(this AProgressBar nativeProgressBar, IProgress progress)
+		{
+			nativeProgressBar.Progress = (int)(progress.Progress * 10000);
```

Comment:
This number is big and scary. Needs to be a constant.

Also, this is 10K and the slider was 1K. Should this be the same, somehow?

Reply:
Created a constant.